bootstrap.sh

```diff
@@ -369,23 +359,5 @@ if [ "$BUILD_PYTHON" == 1 ] ; then
   PYTHONPATH='' $PIP install mysql-connector-python
 fi
 
-#
-# 4. Installation of development related steps e.g. creating Git hooks.
-#
-
-if [ "$BUILD_TESTS" == 1 ] ; then
```

Comment:
I thought these were needed. Is this being done elsewhere?

Reply:
They were failing on my system. I thought the binaries were missing, but it may have just been a mistake. I will see if I can fix it.